tic-tac-toe.js

```diff
@@ -1,7 +1,86 @@
 function TicTacToe() {
-
+  this.board = [[0, 0, 0],
+                [0, 0, 0],
+                [0, 0, 0]
+                ];
+  this.player_1 = new Player(1, "Ham", "🐷");
+  this.player_2 = new Player(2, "Abu", "🐵");
+  this.current_player = this.player_1;
+  this.winning_sequence = null;
+  this.turnCount = 0;
 }
 
-TicTacToe.prototype = {
-  
+function Player(number, name, marker) {
```

Comment:
I think that this function can be outside of the TicTacToe prototype since it represents a different type of object.